platform/viewer/src/routes/Mode/Mode.jsx

```diff
@@ -181,7 +181,16 @@ export default function ModeRoute({
     }
 
     hotkeysManager.setDefaultHotKeys(hotkeys);
-    hotkeysManager.setHotkeys(hotkeys);
+
+    const userPreferredHotkeys = JSON.parse(
+      localStorage.getItem('hotkey-definitions')
+    );
+
+    if (userPreferredHotkeys?.length) {
```

Comment:
why the usage of question mark here?

Reply:
Since there might not be anything in localStorage with key of `hotkey-definitions`